scripts/linux/buildManifest.sh

```diff
@@ -156,9 +157,13 @@ cat $manifest
 
 echo "Done Building And Pushing Docker Images"
 
-curl -Lo /tmp/manifest-tool 'https://edgebuild.blob.core.windows.net/iotedge-win-arm32v7-tools/manifest-tool-linux-amd64' &&
-    chmod +x /tmp/manifest-tool &&
-    /tmp/manifest-tool --debug push from-spec $IGNORE_MISSING $manifest
+
+if [ $WIN_ARM32_ENABLED -eq 1 ]; then
+    curl -Lo /tmp/manifest-tool 'https://edgebuild.blob.core.windows.net/iotedge-win-arm32v7-tools/manifest-tool-linux-amd64' &&
+        chmod +x /tmp/manifest-tool &&
+        /tmp/manifest-tool --debug push from-spec $IGNORE_MISSING $manifest
+fi
+
```

Comment:
reminder: I added this when I wasn't sure if we would keep the winarm32 builds. It has been removed from the build, so I should remove.